### DISCUSSION TITLE
lb4 openapi - @types/node version problem

### DISCUSSION THREAD

**Discussion by lajosf** [loopbackio/loopback-next#8259] [Q&A] [answered]
Hi,
I have dependency problem with Loopback CLI, lb4. When I try to run `lb4 openapi`, then it says I should downgrade my `@types/node` from `^12.20.42` to `^10.17.60` to make it run. If I downgrade the `@types/node` then it works fine. Is there a way to make the generation work without downgrade my `@types/node` version?
My lb4 CLI version is: `@loopback/cli version: 3.0.1`
Thanks in advance!

**Comment by frbuceta** [comment, marked as answer]
This is fixed in the latest version

**Reply by lajosf** [reply]
Hi, thank you, great!
